process.c

```diff
@@ -566,30 +566,22 @@ proc_get_ppid(VALUE _)
  *
  * Document-class: Process::Status
  *
- *  Process::Status encapsulates the information on the
- *  status of a running or terminated system process. The built-in
- *  variable <code>$?</code> is either +nil+ or a
- *  Process::Status object.
- *
- *     fork { exit 99 }   #=> 26557
- *     Process.wait       #=> 26557
- *     $?.class           #=> Process::Status
- *     $?.to_i            #=> 25344
- *     $? >> 8            #=> 99
- *     $?.stopped?        #=> false
- *     $?.exited?         #=> true
- *     $?.exitstatus      #=> 99
- *
- *  Posix systems record information on processes using a 16-bit
- *  integer.  The lower bits record the process status (stopped,
- *  exited, signaled) and the upper bits possibly contain additional
- *  information (for example the program's return code in the case of
- *  exited processes). Pre Ruby 1.8, these bits were exposed directly
- *  to the Ruby program. Ruby now encapsulates these in a
- *  Process::Status object. To maximize compatibility,
- *  however, these objects retain a bit-oriented interface. In the
- *  descriptions that follow, when we talk about the integer value of
- *  _stat_, we're referring to this 16 bit value.
+ *  A Process::Status may contain (and usually does contain)
+ *  information about a system process.
```

Comment:
I'm not sure when `Process::Status` doesn't contain information about a system process?

Reply:
Line 1174 has what I thought of as an "empty" Process::Status object;  not so?

Reply:
I see. While you're technically correct, I think that saying "may contain" is confusing to the reader and it leaves then guessing in what cases this doesn't contain valid information.

Reply:
Fixed.

Reply:
Thank you!